Resources/Prototypes/Catalog/VendingMachines/Inventories/lawdrobe.yml

```diff
@@ -30,3 +30,6 @@
     SpaceCash100: 2
   emaggedInventory:
     CyberPen: 1
+    ClothingUniformCybersunAttorney: 1 # DeltaV
```

Comment:
I would also put the briefcase and attorney clothes into contraband inventory instead of emagged, The briefcase is one of the snazziest items on offer!